# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1232648960 18000
#      Thu Jan 22 13:29:20 2009 -0500
# Node ID da61d0f7ce0b7e3578652b58057883003707bd03
# Parent  e17f262a02cd23afe690dc0e5b8a9bc85481254c
configure.in (AH_BOTTOM): define OCTAVE_USE_OS_X_API if __APPLE__ is defined

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-01-22  John W. Eaton  <jwe@octave.org>
+
+	* configure.in (AH_BOTTOM): Define OCTAVE_USE_OS_X_API if
+	__APPLE__ is defined.
+
 2009-01-22  Jaroslav Hajek  <highegg@gmail.com>
 
 	* configure.in: Fix qrupdate warning message.
 
 2009-01-21  John W. Eaton  <jwe@octave.org>
 
 	* Makeconf.in: Substitute X11_INCFLAGS and X11_LIBS.
 	* configure.in: Use AC_PATH_X to check for X11 header files and
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1982,16 +1982,20 @@ typedef int sig_atomic_t;
 #define OCTAVE_HAVE_POSIX_FILESYSTEM 1
 #endif
 
 /* Define if we expect to have <windows.h>, Sleep, etc. */
 #if defined (__WIN32__) && ! defined (__CYGWIN__)
 #define OCTAVE_USE_WINDOWS_API 1
 #endif
 
+#if defined (__APPLE__)
+#define OCTAVE_USE_OS_X_API 1
+#endif
+
 /* sigsetjmp is a macro, not a function. */
 #if defined (sigsetjmp) && defined (HAVE_SIGLONGJMP)
 #define OCTAVE_HAVE_SIG_JUMP
 #endif
 
 #if defined (__DECCXX)
 #define __USE_STD_IOSTREAM
 #endif
